manager/integration/tests/backupstore.py

```diff
@@ -16,13 +16,102 @@
 from common import is_backupTarget_nfs
 from common import get_longhorn_api_client
 from common import delete_backup_volume
+from common import get_backupstore_url
 
 BACKUPSTORE_BV_PREFIX = "/backupstore/volumes/"
 
-
 TEMP_FILE_PATH = "/tmp/temp_file"
 
 
+@pytest.fixture
+def backupstore_s3(client):
+    set_backupstore_s3(client)
+    yield
+    reset_backupstore_setting(client)
+
+
+@pytest.fixture
+def backupstore_nfs(client):
+    set_backupstore_nfs(client)
+    yield
+    reset_backupstore_setting(client)
+
+
+@pytest.fixture(params=["s3", "nfs"])
```

Comment:
This is a good approach to make sure that we run on nfs and s3 correctly.

Reply:
Thanks

Reply:
Good stuff! I learnt a new thing from you today :D